### PR TITLE
Update tagcompliance.rst

### PR DESCRIPTION
**Pull request by alberttwong**
The new default is "maid_status" and I don't believe you should "start" the instance when it's been unmarked.

### CODE REVIEW

**Comment by alberttwong** [comment]
I have no idea why a documentation change gives you a CI test code failure.

**Comment by kapilt** [comment]
Fwiw doc examples are validated in ci for schema validation, re correctness

**Comment by alberttwong** [comment]
per kapilt: maid status is a historical compatibility thing, you can set the tag name in the policy to change the default.

**Comment by ajkerrigan** [comment]
Thanks @alberttwong , good eye catching the inconsistency in the examples here and thanks for submitting a patch :tada: . Regarding the CI failure... there are doc-related tests, but the failure you hit looks unrelated as far as I can tell :eyes:  .

On the topic of `maid_status` though, as you noted that's a legacy holdover. There was some work to clean up those lingering `maid` references a while back (reference: https://github.com/cloud-custodian/cloud-custodian/issues/779). So with that history in mind it's probably useful to have the doc examples all explicitly refer to `c7n_status`, and perhaps note that `maid_status` is a default.

File: docs/source/aws/examples/tagcompliance.rst [hunk 1]
```diff
@@ -72,10 +72,9 @@ Enforce Tag Compliance
          - "tag:Owner": not-null
          - "tag:CostCenter": not-null
          - "tag:Project": not-null
-         - "tag:c7n_status": not-null
+         - "tag:maid_status": not-null
        actions:
          - unmark
-         - start
```
Comment (ajkerrigan) [inline docs/source/aws/examples/tagcompliance.rst:78]: Hmm fair enough :)

**Comment by alberttwong** [comment]
So when I just followed the default it used maid_status, how do I force it to use the new standard?  Shouldn't it just use the new standard automatically?

**Comment by kapilt** [comment]
`mark-for-op` action supports a tag parameter to specify which tag to use 
https://cloudcustodian.io/docs/aws/resources/aws-common-actions.html#aws-common-actions-mark-for-op

**Comment by ajkerrigan** [comment]
I pushed a couple changes based on this thread and the general confusion around `maid_status`:

- Added a prominent note to the top of the tag compliance example page about the `maid_status` default
- Updated all example policies to use explicit tags rather than relying on that historical fallback

I hope this makes that set of examples clearer @alberttwong without encouraging more use of `maid_status`, but let me know if it leaves any lingering questions/concerns. Thanks for raising the issue.

**Review by ajkerrigan — APPROVED** [review]

**Pull request merged**